composer.json

```diff
@@ -17,12 +17,9 @@
     "require": {
         "php": "~8.0.0 || ~8.1.0 || ~8.2.0",
         "ext-json": "*",
-        "elastic/transport": "^8.4",
+        "elastic/transport": "^8.8",
```

Comment:
Since 8.8 elastic/transport supports basic authentication per node. see https://github.com/elastic/elastic-transport-php/pull/22

Reply:
I assume even if we use 8.8, Elastica will still be compatible with older versions of Elasticsearch 8.x

Reply:
Yes